csu

```diff
@@ -155,92 +220,125 @@ function clean {
   printf "\nDeleting dangling images...\n"
   docker rmi $(docker images -f "dangling=true" -q)
 }
+function clean_help {
+  printf "   clean             Delete unused Docker files.\n"
+}
 
 # Delete all Docker containers and images
 function wipe {
   docker rm $(docker ps -a -q)
   docker rmi $(docker images -q)
 }
+function wipe_help {
+  printf "   wipe              Delete all Docker containers and images.\n"
+}
 
 # View logs
 function logs {
   docker-compose logs
 }
+function logs_help {
+  printf "   logs              View logs.\n"
+}
+
+function process_dev_command() {
+  if [ $# -ne 1 ]
+  then
+    printf "${RED}ERROR: dev command requires one parameter!${NC}\n"
+    helpmenu
+    exit 1
+  else
+    case "$1" in
+      docs)
+        documentation
+        exit
+        ;;
+      logs)
+        logs
+        exit
+        ;;
+      makemigrations)
+        makemigrations
+        exit
+        ;;
+      migrate)
+        migrate
+        exit
+        ;;
+      shell)
+        shell
+        exit
+        ;;
+      static)
+        static
+        static_scratch
+        collect_static
```

Comment:
Was more for future proofing, so that command calls three separate functions. If we wanted to add `dev scratch` later, then it would just call the last two.